pkg/antctl/antctl.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2019 Antrea Authors
```

Comment:
Should we name the file antctl_server if it is mainly for server code?
Of if there are common code for both server and client, should we move server only code to another file?

Reply:
Make sense, moved server related to `server.go`.